src/components/Chat/ActionChat.js

```diff
@@ -40,7 +40,7 @@ export default ActionChat;
 
 const styles = StyleSheet.create({
   container: (active) => ({
-    backgroundColor: active ? COLORS.anonSecondary20 : COLORS.white
+    backgroundColor: active ? COLORS.anonSecondary20 : COLORS.almostBlack
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `COLORS.white` to `COLORS.almostBlack` for the inactive state of the container might significantly alter the appearance. Ensure this color change aligns with the overall design and accessibility guidelines, especially contrast requirements.
```diff
-    backgroundColor: active ? COLORS.anonSecondary20 : COLORS.white
+    backgroundColor: active ? COLORS.anonSecondary20 : COLORS.almostBlack
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->